.lgtm.yml

```diff
@@ -10,9 +10,8 @@ extraction:
     after_prepare:
     - cd "$LGTM_WORKSPACE"
     - mkdir installdir
-    - wget https://github.com/tpm2-software/tpm2-tss/archive/master.tar.gz
-    - tar xf master.tar.gz
-    - cd tpm2-tss-master
+    - git clone https://github.com/tpm2-software/tpm2-tss.git
```

Comment:
maybe add `--depth=1` ?

Reply:
> maybe add `--depth=1` ?

No because then git describe doesn't have enough history to actually describe anything, you just get the sha of the HEAD commit.

Reply:
Makes sense